lib/src/delegates/asset_picker_builder_delegate.dart

```diff
@@ -474,8 +474,8 @@ class DefaultAssetPickerBuilderDelegate
   /// 切换路径时的动画曲线
   Curve get switchingPathCurve => Curves.easeInOut;
 
-  /// [bool] Does picker type enable preview.
-  /// 选择器类型是否启用预览
+  /// [bool] Whether the preview of assets is enabled.
```

Comment:
*The last* nit picking. :P

```suggestion
  /// Whether the preview of assets is enabled.
```